Formula/gotk.rb

```diff
@@ -0,0 +1,48 @@
+class Gotk < Formula
+  desc "Experimental toolkit for assembling CD pipelines the GitOps way"
+  homepage "https://toolkit.fluxcd.io/"
+  url "https://github.com/fluxcd/toolkit.git",
+      tag: "0.0.23",
+      revision: "36ba8f0fcbbbbe13b9a32039f0d3597525177798"
+  head "https://github.com/fluxcd/toolkit.git"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
```

Comment:
Please add `kubectl` as a dependency to all targets

Reply:
Fixed in the replacement commit b11986ebd3fd6a907102d8e9cd31b594f1fe9dda. I had previously omitted it because there are so many different ways to install `kubectl` itself.